app-admin/pprocm/pprocm-1.0-r4.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="PProcM"
+MY_P="${MY_PN}-${PV}"
```

Comment:
Looks like `MY_P` isn't used. (Sorry, missed this last time.)

Reply:
nice catch, fixed it. Also starting with a capital letter in the `DESCRIPTION` now.